predict_bias.py

```diff
@@ -137,15 +148,13 @@ def column(var):
 if len(sys.argv) > 5:
     file_f = xr.open_dataset(sys.argv[5])
```

Comment:
can we add comments for what argv[5,6,7] are?

Reply:
Yes, I can do this.